dune-project

```diff
@@ -1,3 +1,22 @@
 (lang dune 3.0)
 (name zarith-ppx)
 (using mdx 0.2)
+
+(generate_opam_files true)
+
+(source (github Drup/zarith-ppx))
+(license ISC)
+(authors "Gabriel Radanne")
+(maintainers "Gabriel Radanne <drupyog@zoho.com>")
+
+(package
+ (name zarith-ppx)
+ (synopsis "Literals for Zarith's arbitrary-precision integers and rationals")
+ (description "The Zarith library implements arithmetic and logical operations over
+arbitrary-precision integers. Zarith-ppx add support for literals
+of the form '13z' and '1.5q'.")
+ (tags (ppx zarith))
+ (depends
+  ppxlib
+  zarith
+  (mdx (and :with-test (>= 1.9.0))))) ; according to dune docs: "Version 0.2 of the stanza requires mdx 1.9.0."
```

Comment:
There is no more bounds on ocaml. Do you think it's better to keep it implicit ?

Reply:
I don't have a strong preference either way, so I can make it explicit again if you wish. I think I just didn't keep it since this no longer directly interfaces with `ocaml`.

Reply:
Fair enough.